package.json

```diff
@@ -19,6 +19,7 @@
     "@radix-ui/react-popover": "^1.1.1",
     "@radix-ui/react-slot": "^1.1.0",
     "@radix-ui/react-tabs": "^1.1.0",
+    "@radix-ui/react-toast": "^1.2.1",
```

Comment:
You added this dependency, but I can't see place where it's used?

If we don't need `react-toast` let's remove it.